doc/user_guide/signal2d.rst

```diff
@@ -32,3 +32,91 @@ method. The parameters `ramp_x` and `ramp_y` dictate the slope of the ramp in `x
 while the offset is determined by the `offset` parameter. The fulcrum of the linear ramp is at the origin
 and the slopes are given in units of the axis with the according scale taken into account.
 Both are available via the :py:class:`~.axes.AxesManager` of the signal.
+
+Peak finding
+------------
+
+.. versionadded:: 1.0.0
+
+The :py:meth:`~.signal.Signal2DTools.find_peaks2D` method provides access to a
+number of algorithms for that achieve peak finding in two dimensional signals.
+The methods available are as follows:
+
+Zaeferrer peak finder
+^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: python
+
+    >>> s.find_peaks2D(method='zaefferer')
+
+This algorithm was developed by Zaefferer [1]_ and the
+implementation here is after the description of the algorithm in the Ph.D.
+thesis of Thomas A. White. It is based on a gradient threshold followed by a
+local maximum search within a square window, which is moved until it is
+centered on the brightest point, which is taken as a peak if it is within a
+certain distance of the starting point.
+
+Ball statistical peak finder
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: python
+
+    >>> s.find_peaks2D(method='stat')
+
+Developed by Gordon Ball, and described in the Ph.D. thesis of Thomas A.
+White, this method is based on finding points which have a statistically
+higher value than the surrounding areas, then iterating between smoothing and
+binarising until the number of peaks has converged. This method is slow, but
+very robust to a variety of image types.
+
+Matrix based peak finding
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: python
+
+    >>> s.find_peaks2D(method='laplacian_of_gaussians')
+    >>> s.find_peaks2D(method='difference_of_gaussians')
+
+These methods are essentially wrappers around the
```

Comment:
It might be good to make that clearer then, in case any pre- or post-processing causes differences in the results.